File: week2/fizzbuzzkata/fizzbuzzkata.py
def FizzBuzz(val):
    if (val == 1):
        return '1'
    elif (val == 2):
        return '2'
    elif (val % 3 == 0 and val % 5 == 0):
        return 'FizzBuzz'
    elif (val % 3 == 0):
        return 'Fizz'
    elif (val % 5 == 0):
        return 'Buzz'

File: week6/fizzbuzzkata/fizzbuzzkata.js
const fizzbuzz = val => {

    return (val % 3 === 0) && (val % 5 === 0) ? 'FizzBuzz'
        : val % 3 === 0 ? 'Fizz'
        : val %  5 === 0 ? 'Buzz'
        : val === 1 ? '1'
        : val === 2 ? '2'
        : val === 3 ? 'Fizz'
        : val === 5 ? 'Buzz'
        : 'Incorrect'
}

export default fizzbuzz